testers/python/specs.json

```diff
@@ -1,5 +1,5 @@
 {
-  "path_to_uam": "/path/to/uam",
+  "path_to_uam": "/home/vagrant/markus-autotesting/testers/python/uam-git",
```

Comment:
This slipped in. I know it's a pain to remember not to commit it.

Reply:
oops